### PR TITLE
Adding meta description and title

### PR DESCRIPTION
**Pull request by sethdaily**
As per email subject: Next.js Consulting Page SEO Description

### CODE REVIEW

File: content/consulting/nextjs.mdx [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
+        Next.js APIs can optionally be deployed to the edge, meaning that API
+        functionality can be run on a server close to the end-user, reducing
+        latency.
     - image: /images/benefits/ux.png
       title: REACT SERVER COMPONENTS
       description: >
-        React Server Components means you can now run code with a Node.js runtime 
-        within your React components.
+        React Server Components means you can now run code with a Node.js
+        runtime  within your React components.
```
Comment (camillars) [inline content/consulting/nextjs.mdx]: 1. Remove extra space after 'runtime'

File: content/consulting/nextjs.mdx [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
     - image: /images/benefits/self-service.png
       title: AUTOMATIC OPTIMISATION
       description: >
-        Next.js features built-in automatic optimisation of images, fonts, and CSS, 
-        resulting in major performance improvements for users.
+        Next.js features built-in automatic optimisation of images, fonts, and
+        CSS,  resulting in major performance improvements for users.
```
Comment (camillars) [inline content/consulting/nextjs.mdx]: 2. remove extra space after 'css,'

File: content/consulting/nextjs.mdx (hunk 1 above)
Comment (sethdaily) [reply content/consulting/nextjs.mdx]: done

File: content/consulting/nextjs.mdx (hunk 2 above)
Comment (sethdaily) [reply content/consulting/nextjs.mdx]: done

**Review by camillars — APPROVED** [review]

**Pull request merged**